AdaptiveCards/sdk/rendering-cards/xamarin/android/host-config.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Host config - Xamarin.Android SDK
+author: almedina-ms
+ms.author: almedina
+ms.date: 12/02/2019
+ms.topic: article
+---
+
+# Host config - Android
+
+To customize the renderer you provide an instance of the HostConfig object. (See [Host Config Schema](../../../rendering-cards/host-config.md) for the full description.)
```

Comment:
This URL seems to be failing on the test site: https://review.docs.microsoft.com/en-us/adaptive-cards/sdk/rendering-cards/xamarin/android/host-config?branch=user%2Falmedina-ms%2FXamarinAndroidDoc